lib/Command/GetTagIdCommand.php

```diff
@@ -40,7 +40,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 		} catch (TagNotFoundException $e) {
 			$logger->error("Tag not found: ".$tagName." ".$e->getMessage());
 		}
-		$logger->info("tag: ".$tag->getId());
+        if (isset($tag)) {
+            $logger->info("tag: ".$tag->getId());
+        }
```

Comment:
Could be undefined on calling

Reply:
yeah, but its better to return 1 in the catch, so its an actual shell error. Did adjust that in my pr